src/main.rs

```diff
@@ -865,6 +877,26 @@ async fn run_with_tracing() -> Result<(), Error> {
     run().await
 }
 
+async fn report_eip_quota_status(
+    ec2_client: &Ec2Client,
```

Comment:
@benesch sadly no such constant in the SDK, but I did update the PR to define our own constant so it's a bit less magical.